chatbot.py

```diff
@@ -136,9 +136,10 @@
         engine.say(w.get_temperature('celsius'))
         engine.runAndWait()
     elif translate in var3:
+        n1 = datetime.datetime.now()
```

Comment:
please use better variable name. like current_time

Reply:
Done! Updated the code.